docs/GettingStarted.md

```diff
@@ -7,63 +7,188 @@ permalink: docs/getting-started.html
 next: tutorial
 ---
 
-Consider a scenario where you want to test the following `sum.js` file:
+Let's get started by writing a test for a hypothetical `sum.js` file:
 
 ```javascript
-// sum.js
-function sum(value1, value2) {
-  return value1 + value2;
+function sum(a, b) {
+  return a + b;
 }
 module.exports = sum;
 ```
 
-We can get up and running with the following 4 steps:
+Create a directory `__tests__/` with a file `sum-test.js`:
 
-1. Create a directory `__tests__/` with a file `sum-test.js`
+```javascript
+jest.unmock('../sum'); // unmock to use the actual implementation of sum
+
+describe('sum', () => {
+  it('adds 1 + 2 to equal 3', () => {
+    const sum = require('../sum');
+    expect(sum(1, 2)).toBe(3);
+  });
+});
+```
+
+Run `npm install --save-dev jest-cli`.
+
+Add the following to your `package.json`:
+
+```js
+"scripts": {
+  "test": "jest"
+}
+```
+
+Run `npm test`:
+
+```
+[PASS] __tests__/sum-test.js (0.010s)
+```
+
+The code for this example is available at
+[here](https://github.com/facebook/jest/tree/master/examples/getting_started).
```

Comment:
It might read a little more naturally as "is available [here](https://github.com/facebook/jest/tree/master/examples/getting_started)" or "is available at [examples/getting_started](https://github.com/facebook/jest/tree/master/examples/getting_started)", what do you think?